docs/UserGuide.md

```diff
@@ -156,7 +156,7 @@ Format: `create n/NAME p/PHONE_NUMBER [e/EMAIL] [a/ADDRESS] [ds/DESCRIPTION] [nw
 | **`p/PHONE_NUMBER`** | `p/[phone_number]`              | Phone Number should only contain numbers and should be at least 3 digits long                   |
 | **`e/EMAIL`**        | `e/[local_part]@[domain]`       | Local Part should only contain alphanumeric characters and these special characters +_.-        |
 | **`a/ADDRESS`**      | `a/[address]`                   |                                                                                                 |
-| **`d/DESCRIPTION`**  | `d/[description]`               |                                                                                                 |
+| **`ds/DESCRIPTION`** | `ds/[description]`              |                                                                                                 |
 | **`nw/NETWORTH`**    | `nw/$[net_worth]`               | Net worth should start with a $ sign, only contain numbers and should be at least 4 digits long |
 | **`mt/TIME`**        | `mt/[DD]-[MM]-[YYYY]-[HH]:[MM]` | DD: 01-31<br/>MM: 01-12<br/>YYYY: 2000-2099<br/>HH:MM: 00:00 - 23:59                            |
 | **`t/TAG`**          | `t/[tag]`                       | Tag should only be `SECURED` or `POTENTIAL`                                                     |
```

Comment:
Are we not doing the <> thing for data fields? Can just merge this one and then ill go fix it afterwards